src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -11,6 +11,7 @@ export type UserKeys = Readonly<{
     publicKey: PublicKey<Uint8Array>;
```

Comment:
There should be unit test updates to this file to assert about the changes you're making.